packages/features/calendars/DestinationCalendarSelector.tsx

```diff
@@ -44,31 +50,63 @@ const DestinationCalendarSelector = ({
     return {};
   };
 
+  const SingleValue = (props) => {
+    const { label, subtitle } = props.data;
+    return (
+      <components.SingleValue {...props} className="flex space-x-1">
+        <p>{label}</p> <p className=" text-neutral-500">{subtitle}</p>
+      </components.SingleValue>
+    );
+  };
+
+  const Option = (props) => {
+    const { label } = props.data;
+    return (
+      <components.Option {...props}>
+        <span>{label}</span>
+      </components.Option>
+    );
+  };
```

Comment:
Use custom components for select to render which integration the calendar is from